state/presence/presence.go

```diff
@@ -313,7 +320,7 @@ func (w *Watcher) flush() {
 // handle deals with requests delivered by the public API
 // onto the background watcher goroutine.
 func (w *Watcher) handle(req interface{}) {
-	logger.Tracef("got request: %#v", req)
+	logger.Tracef("[%s] got request: %#v for model", w.modelUUID[:6], req)
```

Comment:
how about adding a field with the model UUID prefix instead of slicing at each call site? Better yet, what about a logging method which logs at trace and automatically adds the model UUID prefix?